data/korath/korath missions.txt

```diff
@@ -257,3 +257,570 @@ mission "Small Scale Delivery"
+			`	There's definitely something bigger going on, and it's time to track down Iris and get some answers. You last saw her on <planet>.`
+				accept
+
+
+mission "Korath [F05]: The Rainbow Feather"
+	landing
+	invisible
+	source "La'ab Bamen"
+	to offer
+		has "Korath [F04]: Fishy Reasoning: offered"
+	on offer
+		fail
+		conversation
+			`You find Iris in one of the narrow access tunnels of La'ab Bamen, observing the fish behind the glass.`
+			`	"I was wondering when you'd seek me out," she says, with a worried look. "You gave those two patrols quite a scare on Far'en Lei. They had never seen a human before, and they found one armed at their jail's lock."`
+				goto questions
+
+			label check
+			branch compassion
+				has "Korath [F05]: The Rainbow Feather: feather"
+				has "Korath [F05]: The Rainbow Feather: exile"
+				has "Korath [F05]: The Rainbow Feather: freedom"
+
+			label questions
+			choice
+				`	"I meant no harm."`
+					goto harm
+					to display
+						not "Korath [F05]: The Rainbow Feather: know"
+				`	"How do you know about that?"`
+					goto know
+					to display
+						not "Korath [F05]: The Rainbow Feather: know"
+				`	"You seem to know the Korath well. Who are they, really?"`
+					goto feather
+					to display
+						has "Korath [F05]: The Rainbow Feather: know"
+						not "Korath [F05]: The Rainbow Feather: feather"
+				`	"Why did you call Far'en Lei 'their jail's lock'?"`
+					goto exile
+					to display
+						has "Korath [F05]: The Rainbow Feather: know"
+						not "Korath [F05]: The Rainbow Feather: exile"
+				`	"What is the nature of this fish trade?"`
+					goto freedom
+					to display
+						has "Korath [F05]: The Rainbow Feather: know"
+						not "Korath [F05]: The Rainbow Feather: freedom"
+
+			label harm
+			action
+				set "Korath [F05]: The Rainbow Feather: know"
+			`	"The Korath told me what you did: you brought a weapon to Far'en Lei, Captain." She looks deeply concerned. "That, alone, is terrible harm, and you should never do it again. They let you go because it is a planet of peace."`
+				goto check
+
+			label know
+			action
+				set "Korath [F05]: The Rainbow Feather: know"
+			`	"We are allies of the Korath. Naturally, if a human somehow obtains a jump drive and wanders into Korath space, they ask us about it. Landing on Far'en Lei with a weapon does terrible harm. They let you go because it is a planet of peace."`
+				goto check
+
+			label feather
+			action
+				set "Korath [F05]: The Rainbow Feather: feather"
+			`	She removes her necklace and cups the rainbow feather tenderly in her hands. "This was a gift from a Korath woman who is my... close friend. Korath keep memories in their skin and pass knowledge by eating another's memories. The stronger the memory, the more colorful the feathers. This feather has the memories and emotions that she identifies with me.`
```

Comment:
No criticism here, liz-bians are a perfect addition

Reply:
It is intentionally vague, to let the reader fill in the details.

Lizards are gorgeous, though. I'd date one, if it was sentient.